README.md

```diff
@@ -501,6 +510,9 @@ Note: If all the nodes in the groups are not updated to the same MarkLogic versi
 1. If there is a requirement to upgrade MarkLogic version and MarkLogic Kubernetes Helm Chart version both at once, we recommend following upgrade Steps 1 to 5 from [Upgrading MarkLogic Version in your release](#upgrading-markLogic-version-in-your-release) and Steps 1 to 3 from [Upgrading MarkLogic Version in your release](#upgrading-markLogic-version-in-your-release).
 
 2. Once the `helm upgrade` command is executed for both, initiate terminating pods. As recommended, delete the pod-0 i.e. running MarkLogic Bootstrap host first followed by other pods. 
+```
+kubectl delete dnode-group-marklogic-0
```

Comment:
Same here, need to be kubectl delete pod dnode-group-marklogic-0

Reply:
Fixed command